dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/CycleBreakerGraphTransformer.java

```diff
@@ -33,6 +34,9 @@
  */
 final class CycleBreakerGraphTransformer implements DependencyGraphTransformer {
 
+  // DependencyNode inherits Object's equality. No two instances are equal to each other.
```

Comment:
True for now but this is risky. DependencyNode is an interface, not a class. This could change.

Reply:
Good point. Updated to use a set based on IdentityHashMap.